src/openrct2/scripting/bindings/ride/ScRide.cpp

```diff
@@ -325,7 +325,7 @@ namespace OpenRCT2::Scripting
             auto numPrices = std::min(value.size(), ride->GetNumPrices());
             for (size_t i = 0; i < numPrices; i++)
             {
-                ride->price[i] = static_cast<money64>(value[i]);
+                ride->price[i] = std::clamp<money64>(value[i], 0.00_GBP, 20.00_GBP);
```

Comment:
There should be a constant for the maximum price. (If there isn’t, there should be :-P)